sui/token_bridge/sources/complete_transfer.move

```diff
@@ -63,12 +115,55 @@ module token_bridge::complete_transfer {
         )
     }
 
-    /// `verify_and_bridge_out` is only friendly with this module and the
-    /// `complete_transfer` module. For inbound transfers, the deserialized
-    /// transfer message needs to be validated.
+    /// After a transfer is authorized, a relayer contract may unpack the
+    /// `RelayerReceipt` using this method. Coin representing the relaying
+    /// incentive from this receipt is returned. This method is meant to be
+    /// simple. It allows for a coordination with calling `authorize_upgrade`
+    /// before a method that implements `redeem_relayer_payout` in a transaction
+    /// block to consume this receipt.
     ///
-    /// This method also de-normalizes the amount encoded in the transfer based
-    /// on the coin's decimals.
+    /// NOTE: Integrators of Token Bridge collecting relayer fee payouts from
+    /// these token transfers should be calling only this method from their
+    /// contracts. This method is not  guarded by version control (thus not
```

Comment:
There's an extra space between "not" and "guarded".

Reply:
I'll fix this in another PR